integrasjonspunkt/src/test/java/no/difi/meldingsutveksling/cucumber/TimeToLiveValidationSteps.java

```diff
@@ -0,0 +1,120 @@
+package no.difi.meldingsutveksling.cucumber;
+
+import cucumber.api.java.en.Given;
+import no.difi.meldingsutveksling.domain.sbdh.*;
+import no.difi.meldingsutveksling.nextmove.DpoMessage;
+
+import java.time.ZonedDateTime;
+
+import static no.difi.meldingsutveksling.domain.sbdh.SBDUtil.isExpired;
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * checking if expectedResponseDateTime has expired which means the message should not be handled further.
+ * ExpectedResponseDateTime is a value set in the Standard Business Document Header to indicate how long this message
+ * will be allowed to attempt being delivered.  If this time ever expires the validation will catch it.
+ *
+ * isCurrentTimeAfterExpectedResponseDateTime checks the
+ * isExpectedReponseDateTimeExpired
+ *
+ * @Author Jaflaten
+ */
+
+public class TimeToLiveValidationSteps {
+    private StandardBusinessDocument sbd = getDocument();
+    private StandardBusinessDocument sbd2 = getDocument2();
+
+    @Given("^I have obtained expectedResponseDateTime and currentDateTime$")
+    public void isCurrentTimeAfterExpectedResponseDateTime() {
```

Comment:
Hva er caset for den testen her? Den sammenlikner bare to datoer som tilfeldigvis kommer fra to forskjellige sbd'er.

Reply:
Caset står i feature fila. /test.../cucumber. Men det var eit forsøk på en unittest

Reply:
Fiksa ved å fjerne den eine metoden sidan isExpectedResponseDateTImeExpired dekka det begge gjorde.